config/clock.rb

```diff
@@ -0,0 +1,10 @@
+require "clockwork"
+require_relative "boot"
+require_relative "environment"
+
+module Clockwork
+  every(1.day, "scheduled_messages.send", at: "19:30", tz: "UTC") do
+    puts "Sending scheduled messages"
+    MessageSender.new.delay.run
```

Comment:
@jessieay This should be `MessageSender.new.run`. With the `delay` method, the test passes, but the message doesn't actually send.